cellbase-lib/src/main/java/org/opencb/cellbase/lib/variant/annotation/VariantAnnotationCalculator.java

```diff
@@ -676,15 +683,35 @@ private List<VariantAnnotation> runAnnotationProcess(List<Variant> normalizedVar
         if (futurePharmacogenomicsAnnotator != null) {
             futurePharmacogenomicsAnnotator.processResults(pharmacogenomicsFuture, variantAnnotationList);
         }
+        contextAnnotator.processResults(contextFuture, variantAnnotationList);
 
         // Not needed with newCachedThreadPool
         // fixedThreadPool.shutdown();
 
+        // ACMG, only if consequence type is required
+        if (annotatorSet.contains("consequenceType")) {
```

Comment:
All these "include/exclude" accepted values should be defined and centralised somewhere, instead of having hardcoded values